packages/cli/src/services/redis/redis.types.ts

```diff
@@ -0,0 +1,19 @@
+export type RedisClient = n8nRedisClient | BullRedisClient;
```

Comment:
I believe this is the redis client _type_ (or name?), not the client itself. Maybe we should rename this to `RedisClientType` and `N8nRedisClientType` and `BullRedisClientType` (or `*Name`?)